src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessageBean.java

```diff
@@ -128,9 +128,9 @@ public void onMessage(Message message) {
                             IngestReport errorReport = new IngestReport();
                             errorReport.setFailure();
                             if (ex.getMessage() != null) {
-                                errorReport.setReport("Ingest succeeded, but failed to save the ingested tabular data in the database: " + ex.getMessage());
+                                errorReport.setReport(BundleUtil.getStringFromBundle("file.ingest.saveFailed.detail.message") + ex.getMessage());
                             } else {
-                                errorReport.setReport("Ingest succeeded, but failed to save the ingested tabular data in the database; no further information is available");
+                                errorReport.setReport(BundleUtil.getStringFromBundle("file.ingest.saveFailed.message"));
```

Comment:
For this one I used...

```
     public boolean ingestAsTabular(Long datafile_id) {
+        if (true) {
+            throw (new RuntimeException());
+        }
         DataFile dataFile = fileService.find(datafile_id);
```

... to get this screenshot, which also looks fine to me:

![Screen Shot 2023-05-26 at 2 54 47 PM](https://github.com/IQSS/dataverse/assets/21006/279a6cce-b812-4675-a53c-1e316468a0db)